src/style.css

```diff
@@ -0,0 +1,63 @@
+* {
+  margin: 0;
+}
+
+.content {
+  display: flex;
+  flex-direction: column;
+  padding: 32px 22px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url("images/star.svg");
+  background-position: center;
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  margin-left: 4px;
+  flex-shrink: 0;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 > .stars__star:nth-child(0) {
+  background-image: url("images/star-active.svg");
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars--1 > .stars__star:first-child {
+  background-image: url("images/star-active.svg");
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars--2 > .stars__star:nth-child(-n + 2) {
+  background-image: url("images/star-active.svg");
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars--3 > .stars__star:nth-child(-n + 3) {
+  background-image: url("images/star-active.svg");
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars--4 > .stars__star:nth-child(-n + 4) {
+  background-image: url("images/star-active.svg");
```

Comment:
you can join it using the comma

Reply:
Hello @etojeDenys , do you mean to list all classes (stars--n) in one row?
Each of them has a different number for the star.